test/fixtures/poro.rb

```diff
@@ -309,11 +309,27 @@ class CachingCommentSerializer < CommentSerializer
     cache expires_in: 1.day, skip_digest: true
   end
 
-  class CachingModelSerializer < ModelSerializer
+  class CachingModelSerializer < ActiveModel::Serializer
```

Comment:
Inheriting from `ModelSerializer` will make it inherit the associations definitions, and AMS currently does not support redefining associations, hence new associations with the same name will be added.